nvflare/job_config/fed_job_config.py

```diff
@@ -186,6 +186,26 @@ def _get_server_app(self, config_dir, custom_dir, fed_app):
 
         self._copy_ext_scripts(custom_dir, fed_app.server_app.ext_scripts)
         self._copy_ext_dirs(custom_dir, fed_app.server_app)
+        self._copy_file_sources(custom_dir, fed_app.server_app.file_sources)
+
+    def _copy_file_sources(self, custom_dir, file_sources):
+        for s in file_sources:
+            # s is a tuple of (src_path, dest_dir)
+            src_path, dest_dir = s
+            if dest_dir:
+                dest_path = os.path.join(custom_dir, dest_dir)
+                if not os.path.exists(dest_path):
```

Comment:
I think We don't need this if we already pass "exist_ok=True"?

Reply:
But we need this for copying file.